assets/multiqc_config.yml

```diff
@@ -356,6 +357,17 @@ table_columns_name:
     reads_mapped: "Nr. Mapped Reads"
     reads_mapped_percent: "% Mapped Reads"
 
+## Allow collapsing of file names with _R1/_R2 or _1/_2 at the end
+table_sample_merge:
+  "Read 1":
+    - "_R1"
+    - type: regex
+      pattern: "[_.-][rR]?1$"
+  "Read 2":
+    - "_R2"
+    - type: regex
+      pattern: "[_.-][rR]?2$"
+
```

Comment:
Smart!!! Is it possible to share the MultiQC report? I am very curious how the new one looks like.

Reply:
HAha, I just copied from the MultiQC docs from the new functionality page, but sure!

It's still not perfect - if I removed `_raw` it would collapse everything and not have the expansion thingy

[multiqc_.zip](https://github.com/user-attachments/files/17129227/multiqc_.zip)

Reply:
Agree. It's better to keep `_raw`. 
I didn't see `Nonpareil` in the MultiQC report.